torch/_export/serialize.py

```diff
@@ -145,6 +183,22 @@ def export_operator(target, version) -> Operator:
         return Operator(name=str(target), version=version)
 
 
+def import_operator(serialized_target, op_version):
+    if serialized_target.version != op_version:
+        raise SerializeError(
+            f"Target {serialized_target.name} had op version {serialized_target.version} "
+            f"but the existing op version is now {op_version}"
+        )
+    target = torch.ops
+    for name in serialized_target.name.split("."):
+        if not hasattr(target, name):
+            log.warning(f"Could not find operator {target.name}. Returning target as string.")  # noqa: G004
+            return target.name
```

Comment:
Not sure I follow the logic here...

Reply:
Sorry, I meant to just return the original string target here.